src/test/java/ua/softserveinc/tc/validator/EventValidatorOldTest.java

```diff
@@ -1,312 +1,312 @@
-        Set<Integer> recurrentDays = new HashSet<>();
-        recurrentDays.add(1);
-        monthlyEventDto.setDaysOfTheMonth(recurrentDays);
-        Calendar end = Calendar.getInstance();
-        DateFormat sdf = new SimpleDateFormat(ValidationConstants.DATE_FORMAT);
-        monthlyEventDto.setEndTime(sdf.format(end.getTime()));
-        eventValidatorOld.validate(monthlyEventDto, errorsForMonthly);
-        Assert.assertTrue("at least one day later",errorsForMonthly.hasErrors());
-        Assert.assertEquals(errorsForMonthly.getFieldError(ValidationConstants.MONTH_RECURRENT_DAYS).getCode(),
-                ValidationConstants.EVENT_RECCURRENT_END_MUST_BIGER_ONE_DAY_MSG);
-    }
-
-    @Test
-    public void testMonthlyEventDaysEmpty() {
-        Set<Integer> recurrentDays = new HashSet<>();
-        monthlyEventDto.setDaysOfTheMonth(recurrentDays);
-        eventValidatorOld.validate(monthlyEventDto, errorsForMonthly);
-        Assert.assertTrue("Has days for recurrent",
-                errorsForMonthly.hasFieldErrors(ValidationConstants.MONTH_RECURRENT_DAYS));
-        Assert.assertEquals("empty list", ValidationConstants.NO_DAYS_FOR_RECURRENT_EVENT,
-                errorsForMonthly.getFieldError(ValidationConstants.MONTH_RECURRENT_DAYS).getCode());
-    }
-
-    @Test
-    public void testWeeklyEventDays() {
-        eventValidatorOld.validate(recurrentEventDto, errorsForWeekly);
-        Assert.assertTrue(!errorsForWeekly.hasErrors());
-        recurrentEventDto.setDaysOfWeek("");
-        eventValidatorOld.validate(recurrentEventDto, errorsForWeekly);
-        Assert.assertTrue(errorsForWeekly.hasErrors());
-        Assert.assertEquals("empty list", ValidationConstants.NO_DAYS_FOR_RECURRENT_EVENT,
-                errorsForWeekly.getFieldError(ValidationConstants.WEEK_RECURRENT_DAYS).getCode());
-    }
-
-    @Test
-    public void isEventValid() {
-        Assert.assertTrue(eventValidatorOld.isSingleValid(eventDto));
-        eventDto.setStartTime("28-12-2016");
-        Assert.assertFalse(eventValidatorOld.isSingleValid(eventDto));
-        eventDto.setStartTime("notValidForTime");
-        Assert.assertFalse(eventValidatorOld.isSingleValid(eventDto));
-
-        Assert.assertTrue(eventValidatorOld.isReccurrentValid(recurrentEventDto));
-        String tempEndTime = recurrentEventDto.getEndTime();
-        recurrentEventDto.setEndTime(recurrentEventDto.getStartTime());
-        recurrentEventDto.setStartTime(tempEndTime);
-        Assert.assertFalse(eventValidatorOld.isReccurrentValid(recurrentEventDto));
-        recurrentEventDto.setStartTime("notValidTime");
-        Assert.assertFalse(eventValidatorOld.isReccurrentValid(recurrentEventDto));
-    }
-
-    @Test
-    public void supportTest() {
-        Assert.assertTrue(eventValidatorOld.supports(Event.class));
-        Assert.assertFalse(eventValidatorOld.supports(roomService.getClass()));
-    }
-}
+//package ua.softserveinc.tc.validator;
```

Comment:
It is very bad practice comment like that)) Please enclose it as whole with /\*...\*/

i.e. ->

Bad:
//bla-bla-bla
//bla-bla-bla
//bla-bla-bla
//bla-bla-bla

Good:
/*
bla-bla-bla
bla-bla-bla
bla-bla-bla
bla-bla-bla
*/